db/structure.sql

```diff
@@ -8,6 +8,20 @@ SET check_function_bodies = false;
 SET client_min_messages = warning;
 SET row_security = off;
 
+--
+-- Name: plpgsql; Type: EXTENSION; Schema: -; Owner: -
+--
+
+CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
```

Comment:
@hackmastera pg_catalog seems strange...  This does not show up when I run a migrate.  I wonder if it is a remanent from another project?